packages/web/stores/root.ts

```diff
@@ -233,12 +234,7 @@ export class RootStore {
       this.queriesStore,
       this.chainStore.osmosis.chainId,
       makeLocalStorageKVStore("nonibc_transfer_history"),
-      [
-        new AxelarTransferStatusSource(
-          IS_TESTNET ? "https://testnet.axelarscan.io" : undefined,
-          IS_TESTNET ? "https://testnet.api.axelarscan.io" : undefined
-        ),
-      ]
+      [new AxelarTransferStatusSource(), new SquidTransferStatusSource()]
```

Comment:
Seems I made the right call with this approach, feels good. Extensible

Reply:
Yes, it makes it super easy to track txs. Thank you!